crates/primitives/src/transaction/tx_type.rs

```diff
@@ -103,6 +103,15 @@ impl TryFrom<u8> for TxType {
     }
 }
 
+impl TryFrom<U64> for TxType {
```

Comment:
can we to TryFrom<u64> as well?

then TryFrom<U64> is just `value.to::<u64>().try_into`

Reply:
Done